.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
please remove the .idea folder from the repo. add it to the .gitignore so it doesnt appear again